server.go

```diff
@@ -234,3 +237,128 @@ func (b *backend) Move(r *http.Request, dest *internal.Href, overwrite bool) (cr
 	}
 	return created, err
 }
+
+type HomeSet interface {
```

Comment:
This interface looks like it won't be easily re-usable in the DAV client. It would be nicer to find a good name to reflect this.